AEPAssurance/Source/PincodeEntry/SessionAuthorizingUI.swift

```diff
@@ -19,7 +19,7 @@ protocol SessionAuthorizingUI {
     /// property that indicated if the pinCode screen is currently displayed
     var isDisplayed: Bool { get }
 
-    init(withExtension: Assurance)
+    init(withState stateManager: AssuranceStateManager)
```

Comment:
would prefer param label of `withStateManager` over `withState` because it's more explicit